todo_app/app.py

```diff
@@ -1,15 +1,37 @@
-from flask import Flask
-
+from flask import Flask,render_template, request, redirect
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item, save_item, get_item
+
 
+# create flask app
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+# index route 
```

Comment:
This is the kind of code comment that is more likely to clutter the codebase or go out of date, rather than be helpful.